docs/GettingStartedDocs/Contributors/WindowsSGX1FLCGettingStarted.md

```diff
@@ -0,0 +1,131 @@
+# Getting Started with Open Enclave on Windows for systems with support for SGX1 with Flexible Launch Control(FLC)
+
+## Platform requirements
+
+Intel® X86-64bit architecture with SGX1 and Flexible Launch Control (FLC) support. (e.g. Intel Coffee Lake CPU)
+
+Note: To check if your system has support for SGX1 with FLC, please look [here](../SGXSupportLevel.md)
+
+A version of Windows OS with native support for SGX features:
+- For server: Windows Server 2016
+- For client: Windows 10 64-bit version 1709 or newer
+- To check your Windows version, run `winver` on the command line.
+
+## Install Git and Clone the Open Enclave SDK repo
+
+Download and install Git for Windows from [here](https://git-scm.com/download/win)
+
+Clone the Open Enclave SDK
+
+```powershell
+git clone https://github.com/openenclave/openenclave.git
+```
+
+This creates a source tree under the directory called openenclave.
+
+## Install project prerequisites
+
+First, change directory into the openenclave repository:
+
+```powershell
+cd openenclave
+```
+
+Run the following from powershell to deploy all the prerequisites for building Open Enclave (including Intel's DCAP primitives and Azure's DCAP library).
+
+```scripts/install-windows-prereqs.ps1```
+
+To install the prerequisites along with the Azure Data Center Attestation Primitives (DCAP) Client, use the below command. The Azure DCAP Client is necessary to perform attestation on an Azure Confidential Computing VM.
+
+```powershell
+cd scripts
+.\install-windows-prereqs.ps1 -InstallPath YOUR_WORKSPACE_PATH_HERE -LaunchConfiguration SGX1FLC -DCAPClientType Azure
+```
+If you would like to skip the installation of the Azure DCAP Client, use the command below.
+
+```powershell
+cd scripts
+.\install-windows-prereqs.ps1 -InstallPath YOUR_WORKSPACE_PATH_HERE -LaunchConfiguration SGX1FLC -DCAPClientType None
+```
+If there is another DCAP Client type that you would like to install, you are welcome to submit a pull request to add support for that DCAP Client.
```

Comment:
I don't think this script should be maintained to support arbitrary DCAP clients. I think it should be considered a bug that install-windows-prereqs.ps1 understands Azure DCAP client as a special type; anything specific to our configuration for CI/CD should be partitioned into Ansible, and if there is to be a default for this script, it should integrate with Intel's DCAP client/service.

Reply:
It appears that the Intel DCAP Client has not been tested and therefore is not in the pre-reqs script. I have opened Issue#2177 to track this. As of now, I am keeping these instructions so developers have a path to obtain DCAP attestation on ACC VMs. As soon as the Intel DCAP Client is tested, I can change the docs to use Intel DCAP Client instead.